### ISSUE TITLE
No face recognition being performed

### ISSUE THREAD

**Issue by karthikgreninja** [Alpkant/Thermal-to-Visible-Face-Recognition-Using-Deep-Autoencoders#4]
In the paper it was mentioned that face recognition was being performed. However, I could find no face recognition code provided here. Can you please upload the face recognition code as well? Thanks

**Comment by Alpkant** [comment]
The face recognition model checkpoints are not available. You can use any face recognition model to train. Our proposed method tries to convert the RGB images to thermal and vice versa. We try to decrease the domain gap between each modality. Any face recognition model can be used on our model outputs.

**Comment by karthikgreninja** [comment]
Oh ok. Thank you for replying so soon!

**Comment by karthikgreninja** [comment]
The datasets mentioned in the paper are not easily accessible right now. Could you please share the datasets or share the method you used to access the datasets? Thanks

**Comment by Alpkant** [comment]
I don't have the right to redistribution so I can not share the images. However, I can give the links that I used to access the datasets.
For the EURECOM dataset, I used this link: http://vis-th.eurecom.fr/contact
For Carl dataset, I used this link: http://splab.cz/en/download/databaze/carl-database
These datasets are generally open for academic research therefore you will need an advisor to sign the license agreement files.

**Comment by karthikgreninja** [comment]
Any tips on how to improve the accuracy of the UNet generated images? When training and testing on the generated images accuracy values are coming very low. Thanks

**Comment by Alpkant** [comment]
Strong augmentation is recommended. Color augmentations or some rotation and flips will help to increase the accuracy. Also careful finetuning is important.
